integration_tests/reload_api_test.go

```diff
@@ -31,6 +32,34 @@ var _ = Describe("reload API endpoint", func() {
 			Expect(resp.StatusCode).To(Equal(405))
 			Expect(resp.Header.Get("Allow")).To(Equal("POST"))
 		})
+
+		Context("with a non-zero reload interval", func() {
+			newRouterPort := 7999
+			newApiPort := 8000
+			BeforeEach(func() {
+				err := startRouter(newRouterPort, newApiPort, map[string]string{"ROUTER_RELOAD_INTERVAL": "1s"})
```

Comment:
I think a second here is overkill for the tests and will only slow them down unnecessarily.  It should be sufficient to sue something like 100ms or possibly even less.